### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by valt19**
Add a validation form to the contact form

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -645,6 +721,21 @@ h4 {
   margin-left: 15px;
 }
 
+#form-id #errormessage {
+  position: absolute;
```
Comment (didierganthier) [inline style.css:725]: Well done on the error message 👏🏽 but this line is making your error not appear near the form when the email is not fully lowercase, you can either remove this and have it `hidden` by default and toggle it using JavaScript DOM interactions. 🤓 Check my other comment.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+});
+
+const contactForm = document.getElementById('form-id');
+const contactEmail = document.getElementById('email');
+const contactError = document.getElementById('errormessage');
+
+contactForm.addEventListener('submit', (event) => {
+  if (contactEmail.value.toLowerCase() !== contactEmail.value) {
+    event.preventDefault();
+    contactError.innerText = 'please type your email in lowercase.form not submitted';
```
Comment (didierganthier) [inline script.js:232]: Your error message is empty 🗑️  at first but when you set the `innerText` it is always `absolute` so it doesn't show up 👀 you can fix this regardless of my other comment by adding this line below:
```
contactError.style.position = 'inherit';
```
This just gives the `div` the right position attribute to make it show up near the fields. 😉

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @valt19   :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**